views/layouts/layout-example.njk

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ title }}</title>
+    <link href="/stylesheets/main.css" rel="stylesheet" media="all" />
+    <!--[if lt IE 9]><script src="/javascripts/ie.js"></script><![endif]-->
+    <script src="/javascripts/application.js"></script>
+  </head>
+  <body class="govuk-u-font-smoothing app-u-example">
```

Comment:
I've lost track, is the font smoothing class needed because we still need to use the latest version of Frontend? Or was this removed last time?

Reply:
It seems like we have it across all layouts. I'm happy to remove, but not only from this file.

Reply:
It might be something we should remove when we update to use the latest version of Frontend.